rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -6046,12 +6045,9 @@ bool opcode_0207(const ScriptArguments& args, const ScriptCharacter character, c
     @arg arg2 
     @arg arg3 
 */
-void opcode_0208(const ScriptArguments& args, const ScriptFloat arg1, const ScriptFloat arg2, ScriptFloat& arg3) {
-    RW_UNIMPLEMENTED_OPCODE(0x0208);
-    RW_UNUSED(arg1);
-    RW_UNUSED(arg2);
-    RW_UNUSED(arg3);
+void opcode_0208(const ScriptArguments& args, const ScriptFloat low, const ScriptFloat high, ScriptFloat& random) {
     RW_UNUSED(args);
+    random = low + static_cast <float> (std::rand()) / (static_cast <float> (RAND_MAX / (high - low)));
```

Comment:
Here also we should use c++11 random.

Reply:
You are totally righ, but these three have since been taken care of in a tasty manner by @christoph-heiss in #439 ! :smile:

Reply:
In that case, I'd suggest to remove these opcodes from this patch to avoid merge conflicts and changing back and forth.